### PR TITLE
zmaticoo Bid Adapter : add support for video

### PR DESCRIPTION
**Pull request by lxj15398019970**
## Type of change

- [x ] Feature

### CODE REVIEW

File: modules/zmaticooBidAdapter.js [hunk 1]
```diff
@@ -103,28 +163,36 @@ export const spec = {
...
+            ad: zmBid.adm,
+            ttl: TTL,
+            creativeId: zmBid.crid,
+            netRevenue: NET_REV,
+          };
+          if (zmBid.adomain && zmBid.adomain.length) {
+            bid.meta = {
+              advertiserDomains: zmBid.adomain
+            };
+          }
```
[the remark this answers is not in the document]
Comment (lxj15398019970) [reply modules/zmaticooBidAdapter.js]: ok

File: modules/zmaticooBidAdapter.js [hunk 2]
```diff
@@ -103,28 +163,36 @@ export const spec = {
...
-        height: zbid.h,
-        ad: zbid.adm,
-        ttl: TTL,
-        creativeId: zbid.crid,
-        netRevenue: NET_REV
-      };
-      bidResponse.push(bid);
+    let bidResponses = [];
+    const response = (serverResponse || {}).body;
+    if (response && response.seatbid && response.seatbid[0].bid && response.seatbid[0].bid.length) {
```
[the remark this answers is not in the document]
Comment (lxj15398019970) [reply modules/zmaticooBidAdapter.js]: ok

**Comment by osazos** [comment]
Thanks for your update, I just saw your coverage rate is too low, please add tests.

**Review by osazos — APPROVED** [review]
LGTM

**Pull request merged**